icu4c/source/common/umapfile.cpp

```diff
@@ -238,7 +238,12 @@ typedef HANDLE MemoryMap;
         pData->pHeader=(const DataHeader *)data;
         pData->mapAddr = data;
 #if U_PLATFORM == U_PF_IPHONE || U_PLATFORM == U_PF_ANDROID
+    // Apparently supported from Android 23 and higher:
+    //   https://github.com/ggml-org/llama.cpp/pull/3631
+    // But I think that checking for the flag itself is safer than checking for __ANDROID_API__
```

Comment:
Reads like a temporary pull request comment. Let's rephrase for posterity.
```suggestion
    // Checking for the flag itself is safer than checking for __ANDROID_API__.
```